recipes-kernel/linux/linux-hikey_4.5.bb

```diff
@@ -0,0 +1,17 @@
+require linux-hikey.inc
+
+DESCRIPTION = "Hisilicon 4.5 Kernel"
+
+PV = "4.5+git${SRCPV}"
+SRCREV_kernel="${AUTOREV}"
```

Comment:
No AUTOREV allowed, period. Also, if you're doing AUTOREV, why set a PV? It will very soon be wrong.